src/pages/components/menu/style.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Menu
+tabs: ['Usage', 'Style', 'Code', 'Accessibility']
+---
+
+## Color
```

Comment:
Just a general thing for Style tabs, it might be good to start using Aubrey's [new Style tab content template](https://carbondesignsystem.com/contributing/documentation/#style-template) here.